apps/web/src/utils/badge.tsx

```diff
@@ -0,0 +1,55 @@
+import Tooltip from "@web/components/Tooltip";
+import { Permer } from "permer";
+import { Heart, X } from "react-feather";
+import { SelfUser } from "../../../../packages/commons/types";
+
+// temporary icons for testing.
+// TODO: sometimes badges look a little weird when placed
+// next to each other, please look into!
+const badges = [
+  {
+    id: "admin",
+    tooltip: "imperialb.in Staff",
+    icon: <Heart color="red" fill="red" />,
+  },
+
+  {
+    id: "beta-tester",
+    tooltip: "Beta Tester",
+    icon: <Heart color="purple" fill="purple" />,
+  },
+
+  {
+    id: "early-adopter",
+    tooltip: "Early Adopter",
+    icon: <Heart color="orange" fill="orange" />,
+  },
+
+  {
+    id: "member-plus",
+    tooltip: "Supporter of imperialb.in",
+    icon: <X color="orange" />,
```

Comment:
might need to make a custom icon for this one honestly, so just for now set it as the Plus icon.